lib/run_utils.py

```diff
@@ -0,0 +1,61 @@
+import sys
+import os
+import csv
+from lib.prediction_class import *
+
+def split_algorithms(prediction_algorithms):
+    class_i_prediction_algorithms = []
+    class_ii_prediction_algorithms = []
+    if 'all' in prediction_algorithms:
+        class_i_prediction_algorithms = MHCI.prediction_methods()
+        class_ii_prediction_algorithms = MHCII.prediction_methods()
+    elif 'all_class_i' in prediction_algorithms or 'all_class_ii' in prediction_algorithms:
+        if 'all_class_i' in prediction_algorithms:
+            class_i_prediction_algorithms = MHCI.prediction_methods()
+        if 'all_class_ii' in prediction_algorithms:
+            class_ii_prediction_algorithms = MHCII.prediction_methods()
+    else:
```

Comment:
What if I want to specify `all_class_i` plus a single Class II algorithm (or vice versa)? Should that be supported?

Reply:
maybe an easy way would be to create an array of prediction methods, expand the "all" options out to all their components, and then unique the array?  That then handles the case where you specify all and then list some others explicitly